lib/ddtrace/contrib/sneakers/ext.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Datadog
+  module Contrib
+    module Sneakers
+      # Sneakers integration constants
+      module Ext
+        APP = 'sneakers'.freeze
+        ENV_ANALYTICS_ENABLED = 'DD_SNEAKERS_ANALYTICS_ENABLED'.freeze
```

Comment:
small nit: the new convention for analytics_enabled and analytics_sample_rate is to prepend the env var's with `DD_TRACE` instead of `DD_`,  per https://github.com/DataDog/dd-trace-rb/pull/1115

I don't believe we'll need to add support for the deprecated format, but good to have the `DD_SNEAKERS_ANALYTICS_ENABLED` and `DD_TRACE_SNEAKERS_ANALYTICS_SAMPLE_RATE` format for new integrations going forward

Reply:
Hmm good question. I personally would also not add any legacy support, since this will be a new integration none has used.

Reply:
yup sounds good, no need to add the legacy options on sneakers

Reply:
Sadly [some tests in the analytics examples](https://github.com/janz93/dd-trace-rb/blob/master/spec/ddtrace/contrib/analytics_examples.rb#L155) expect them ... :(

In order to ship this PR soon, I added the legacy ENV's.

I would suggest a separate cleanup commit where this will be removed enterly. What do you think? @ericmustin

Reply:
ah, yes good point, yes will remove these in the future. All the settings/env vars lgtm!